lib/plugins/dev-server.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+const pathUtil = require('../config/path-util');
+const AssetOutputDisplayPlugin = require('../friendly-errors/asset-output-display-plugin');
+const FriendlyErrorsPlugin = require('./friendly-errors');
+
+/**
+ * @param {WebpackConfig} webpackConfig
+ * @param {Array} paths to clean
+ * @param {Object} cleanUpOptions
+ * @return {Array} of plugins to add to webpack
+ */
+module.exports = {
+    getPlugins(webpackConfig) {
+        const outputPath = pathUtil.getRelativeOutputPath(webpackConfig);
+        const friendlyErrorsPlugin = FriendlyErrorsPlugin.getPlugins();
+        return [new AssetOutputDisplayPlugin(outputPath, friendlyErrorsPlugin[0])];
```

Comment:
> In which way a singleton will be different to the original code when we call Encore.reset() to generate a new configuration?

Honestly, in practice, it probably makes not difference for `FriendlyErrorsWebpackPlugin`. In fact, it's not clear to me what the proper behavior is for this plugin specifically when it comes to multi webpack config setups - there's an option question about it: https://github.com/geowarin/friendly-errors-webpack-plugin/issues/56.

Could we just pass in the FriendlyErrorsPlugin instance as a dependency - i.e. add a `friendlyErrorsWebpackPlugin` argument to this plugin and pass the instance in from `config-generator`? It would work and no complexity or magic.